rules/crowdstrike_rules/crowdstrike_detection_passthrough.yml

```diff
@@ -63,3 +64,126 @@ Tests:
+        "event": {
+          "AgentIdString": "fa6a04a7f18d473fa06771b4961aa3d9",
+          "CommandLine": "/bin/echo CROWDSTRIKE_SAMPLE_DETECTION",
+          "ComputerName": "hostname.lan",
+          "CustomerIdString": "712bcd164963442ea43d52917cecdecc",
+          "DetectDescription": "Non-malicious sample detection generated for evaluation purposes.",
+          "DetectId": "ldt:fa6a04a7f18d473fa06771b4961aa3d9:346037607921826886",
+          "DetectName": "Suspicious Activity",
+          "EventType": "Event_ExternalApiEvent",
+          "EventUUID": "4624ba0c46c8405ea4998a0df7a5fa53",
+          "ExternalApiType": "Event_DetectionSummaryEvent",
+          "FalconHostLink": "https://falcon.us-2.crowdstrike.com/activity/detections/detail/00000000000000000000000000000000/222222222222222222?",
+          "FileName": "echo",
+          "FilePath": "/bin/",
+          "GrandparentCommandLine": "-bash",
+          "GrandparentImageFileName": "/bin/bash",
+          "LocalIP": "192.168.86.255",
+          "MACAddress": "a4-83-e7-19-28-8b",
+          "MD5String": "2153a89b0a91c38f152acbefafd69b99",
+          "MachineDomain": "",
+          "Nonce": 1,
+          "Objective": "N/A",
+          "OriginSourceIpAddress": "",
+          "ParentCommandLine": "bash",
+          "ParentImageFileName": "/bin/bash",
+          "ParentProcessId": 346017374056168100,
+          "PatternDispositionDescription": "Detection, standard detection.",
+          "PatternDispositionFlags": {
+            "BootupSafeguardEnabled": false,
+            "CriticalProcessDisabled": false,
+            "Detect": false,
+            "FsOperationBlocked": false,
+            "InddetMask": false,
+            "Indicator": false,
+            "KillParent": false,
+            "KillProcess": false,
+            "KillSubProcess": false,
+            "OperationBlocked": false,
+            "PolicyDisabled": false,
+            "ProcessBlocked": false,
+            "QuarantineFile": false,
+            "QuarantineMachine": false,
+            "RegistryOperationBlocked": false,
+            "Rooting": false,
+            "SensorOnly": false
+          },
+          "PatternDispositionValue": 0,
+          "ProcessEndTime": 1616609989,
+          "ProcessId": 346037607908199600,
+          "ProcessStartTime": 1616609989,
+          "SHA1String": "0000000000000000000000000000000000000000",
+          "SHA256String": "9e8a9e9f8cab6f07d8e711043bf856893660143aaf4385cb4e7b6faa97d6e61e",
+          "SensorId": "fa6a04a7f18d473fa06771b4961aa3d9",
+          "Severity": 2,
+          "SeverityName": "Low",
+          "Tactic": "N/A",
+          "Technique": "N/A",
+          "UTCTimestamp": 1616609989000,
+          "UserName": "username",
+          "cid": "712bcd164963442ea43d52917cecdecc",
+          "eid": 118,
+          "timestamp": "2021-03-24T18:19:49Z"
+        },
+        "fdr_event_type": "Event_DetectionSummaryEvent",
+        "filename": "echo",
```

Comment:
These two fields seem as remnants. `FileName` and `FilePath` are defined above already (and they seem correct).